pkg/db/config.go

```diff
@@ -26,6 +26,29 @@ type DatabaseConfig struct {
 	NameFile           string `json:"name_file"`
 	UsernameFile       string `json:"username_file"`
 	PasswordFile       string `json:"password_file"`
+
+	EnablePreparedStatements bool
+}
+
+func (d *DatabaseConfig) DeepCopy() *DatabaseConfig {
+	return &DatabaseConfig{
+		DatabaseCaCertFile:       d.DatabaseCaCertFile,
+		Debug:                    d.Debug,
```

Comment:
:+1:  I was aware of that, but I preferred to individually assign them so that when new fields are added, which might not be primitive types, it is noticeable in comparison (even at the risk of missing adding them in the method)